cmd/cosign/cli/sign/sign.go

```diff
@@ -73,6 +72,7 @@ func ShouldUploadToTlog(ctx context.Context, ko options.KeyOpts, ref name.Refere
 	// Check if TSA signing is enabled and tlog upload is disabled
 	if !tlogUpload && tsaServerURL != "" {
 		fmt.Fprintln(os.Stderr, "\nWARNING: skipping transparency log upload")
+		return false
```

Comment:
Looks like tlog-upload defaults to false, for they keyless signing+rekor case I think we will return false here before we get a chance to check for keyless signing.

https://github.com/sigstore/cosign/blob/44a7117e0cee495a163c92e9f347efb2d9b747c7/cmd/cosign/cli/options/signblob.go#L75

Reply:
For keyless signing + rekor, `tsaServerURL=""` will be empty so it won't enter here.

Reply:
nevermind! i assumed we had a default set for `timestamp-server-url` since we do for fulcio and rekor, but i see now that we don't. please disregard!